OneCommunityGlobal/HighestGoodNetworkApp
# Kaushik - Finished Cheng-Yun Project.jsx Unit Test

# Description
Provide the unit test functions for Project.jsx

## Related PRS (if any):
N/A

## Main changes explained:
- Update Project.jsx for inserting data-test id
- Create Project.test.jsx

## How to test:

1. check into current branch
2. do `npm install`
3. `npm run test src/components/Projects/Project/Project.test.jsx`
4. verify that all tests pass


## Screenshots or videos of changes:
![Screenshot 2023-11-29 at 10 39 12 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/74082492/8074f6c4-931d-49c8-9dda-f107ecd8aed4)

## Note:
